File: 217-contains-duplicate/contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        int i,n;
        Arrays.sort(nums);
        n=nums.length;
        for (i=1;i<n;i++)
        {
            if (nums[i]==nums[i-1])
            {
                return true;
            }
        }
       
            
        return false;

        
    }
}

File: 217-contains-duplicate/contains-duplicate.py
from collections import Counter
from typing import List

class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        count = Counter(nums)
        # If any element appears more than once, return True
        for freq in count.values():
            if freq > 1:
                return True
        return False
